### ISSUE TITLE
Undesirable behaviour when opening a directory

### ISSUE THREAD

**Issue by piotr-machura** [kyazdani42/nvim-tree.lua#646]
When opening a directory with `nvim .` and the `g:nvim_tree_disable_window_picker` option is set neither `<CR>` nor `o` open a file (I guess since the "previously used window" does not exist), when pressing either nothing happens.

The only configuration needed to reproduce is `lua require('nvim-tree').setup{}` and `g:nvim_tree_disable_window_picker` set before it.

The bug is not present on 67805502d2a126a4c40466059db8f1770a6625d4 (before the 1.3.0 refactor)

**Comment by piotr-machura** [comment]
Nevermind, I think I messed something up because it's working now. Perhaps it's because the new default behaviour is to **not** open the tree when directory is opened, since by default `open_on_setup` is false (yet it was still opening without setting the option, but isn't now? Weird.). Or was that always the case and I'm imagining things?
Closing the issue anyways.

**Comment by ybbond** [comment]
In my case, opening a directory will always trigger `nvim-tree`. I setup to not open at startup (or on setup), but this keep happening.

My setup:
```lua
vim.g.nvim_tree_side = 'right'
vim.g.nvim_tree_width = 40
-- vim.g.nvim_tree_auto_open = 0

require'nvim-tree'.setup {
  disable_netrw = false,
  hijack_netrw = false,
  open_on_setup = false,
  open_on_startup = false,
  open_on_tab = false,
  update_cwd = false,
  lsp_diagnostics = true,
  update_focused_file = {
    enable = true,
    update_cwd = false,
  },
  view = {
    width = 40,
    side = 'right',
    auto_resize = false,
  }
}
```

**Comment by kyazdani42** [comment]
fixed in `1.4.7`
